tsl/src/nodes/decompress_chunk/planner.c

```diff
@@ -336,6 +336,35 @@ clause_has_compressed_attrs(Node *node, void *context)
 	return expression_tree_walker(node, clause_has_compressed_attrs, context);
 }
 
+/*
+ * Find the resno of the given attribute in the provided target list
+ */
+static AttrNumber
+find_attr_pos_in_tlist(List *targetlist, AttrNumber pos)
+{
+	ListCell *lc;
+
+	Assert(targetlist != NIL);
+	Assert(pos > 0 && pos != InvalidAttrNumber);
+
+	foreach (lc, targetlist)
+	{
+		TargetEntry *target = (TargetEntry *) lfirst(lc);
+
+		if (!IsA(target->expr, Var))
+			elog(ERROR, "compressed scan targetlist entries must be Vars");
+
+		Var *var = (Var *) target->expr;
+		AttrNumber compressed_attno = var->varattno;
+
+		if (compressed_attno == pos)
+			return target->resno;
+	}
+
+	elog(ERROR, "Unable to locate var %d in targetlist", pos);
+	pg_unreachable();
```

Comment:
pg_unreachable is included into elog(ERROR).

Reply:
This is needed for the compiler on windows. Without `pg_unreachable` it complains that we have to return something from this non-void function. Therefore, I added this statement.